_specifications/specification-3-17.md

```diff
@@ -9176,6 +9183,42 @@ export interface Moniker {
 
 Server implementations of this method should ensure that the moniker calculation matches to those used in the corresponding LSIF implementation to ensure symbols can be associated correctly across IDE sessions and LSIF indexes.
 
+#### <a href="#textDocument_validateBreakableRange" name="textDocument_validateBreakableRange" class="anchor">Validate Breakable Range (:leftwards_arrow_with_hook:)</a>
+
+> *Since version 3.17.0*
+
+The `textDocument/validateBreakableRange` request is sent from the client to the server to adjust a breakpoint or active statement range. The server response is the best corresponding range for the client to use. `null` should be returned if a breakpoint should not be placed near the requested range. If the server is not able to determine a precise range (e.g. due to the range being in a disabled preprocessor block), the server can return a 0 width range located at the beginning of the line.
+
```

Comment:
We need to add a client capability that indicates whether that request support dynamic registration on the client side. You can copy that from another request.